blog/authors.yml

```diff
@@ -1,20 +1,20 @@
 phillebaba:
   name: Philip Laine
-  title: DevOps Engineer
+  title: Expert DevOps Engineer
   url: https://github.com/phillebaba/
-  image_url: https://media-exp1.licdn.com/dms/image/C4E03AQG31KQ2xlivRA/profile-displayphoto-shrink_800_800/0/1598524888194?e=1655337600&v=beta&t=syoZr78rwDPb7Jj4d4B4opGNVCVZIa_3rmCkX-6oT2Q
+  image_url: https://media-exp1.licdn.com/dms/image/C4E03AQG31KQ2xlivRA/profile-displayphoto-shrink_800_800/0/1598524888194?e=1673481600&v=beta&t=lIA0UixRG6tq22FXe9IuxYKjiINqgVgkLw9hWMdqD_w
```

Comment:
We could change these to a `data:` url and just embed the data, if we're afraid they might disappear?

Reply:
Gave it 5 min try, didn't work. To lazy to go on.
Thanks for the input though :)